lib/building.rb

```diff
@@ -1,12 +1,8 @@
-require 'pry'
-
-
 class Building
   attr_reader :units, :occupancy
 
   def initialize
     @units = []
-    @occupancy = 0.0
```

Comment:
We don't need to store @occupancy because we're never updating it. If we just need to return `0.0` whenever the units are empty, we can do that in the calculation below.